demos/restart_demo.sh

```diff
@@ -23,9 +23,12 @@ source utils.sh
 
 install_dependencies () {
     echo "Installing dependencies..."
-    pip install -U pip
-    pip install -U jupyter
-    pip install -U jupyterhub
+    apt install -y npm
+    npm install -g configurable-http-proxy
+    apt install -y python3-pip
+    #pip3 install -U pip
```

Comment:
I realize this was in the original diff, but would you mind removing it since it's just commented out?